appendix.md

```diff
@@ -0,0 +1,26 @@
+# Browser Columns
+
+All browser columns are available for both
+[Cards and Notes mode](browsing.md#table-modes) but sometimes with slightly
+different names and data. The following table lists the behaviours for both modes.
+The list of cards or notes can be sorted by every column except for the
+Question and Answer columns.
+
+
+| Column | Cards mode | Notes mode |
+|-|-|-|
+| Answer | Shows the back side of the card in one line with the question stripped. You can also choose a [custom format](templates/styling.md#browser-appearance) in the card type editor instead. | Shows the same as in Cards mode, only for the first card of the note. |
+| Card(s) | Shows the name of the card's template. | Shows the number of cards the note has. |
+| Card Modified | Shows the last time changes were made to the card (e.g., when you reviewed the card and the review history and interval were updated). | Shows the last time changes were made to one of the note's cards. |
+| Created | Shows the same as in Notes mode for the card's note. | Shows when the note was created. |

```

Comment:
In cases where both are the same, maybe we could we collapse the two columns into 1, and show the text once?

Reply:
That would be great, but I don't think Markdown supports that?

Reply:
Ah, that's a shame :-(